src/components/tags/tags.js

```diff
@@ -26,7 +26,12 @@ export class TagsComponent {
   }
 
   handleAddTagClicked() {
-    this.addTagDialogComponent.showDialog();
+    this.addTagDialogComponent.showDialog(this.tagsValue());
+  }
+
+  tagsValue() {
```

Comment:
this is a method -- so it should be `getTagsValues` -- `values` should be in plural -- because it is an array